src/main/java/frc/robot/RobotContainer.java

```diff
@@ -237,7 +240,13 @@ private void configureBindings() {
     OI.getButton(OI.Operator.retractClimber).toggleOnTrue(climberCommandFactory.climb());
 
     new Trigger(() -> OI.Operator.controller.getPOV() == 0).whileTrue(intakeCommand());
+    // new Trigger(() -> OI.Operator.controller.getPOV() == 270)
+    //     .onTrue(new InstantCommand(() -> robotStateManager.setLongRange()));
```

Comment:
Let's actually remove this code, and the setShortRange

Reply:
resolved